### PR TITLE
RUMM-2387 Rename Crash Reporting podspec

### PR DESCRIPTION
**Pull request by maxep**
### What and why?

Rename `DatadogSDKCrashReporting.podspec` to `DatadogCrashReporting.podspec` and mark `DatadogSDKCrashReporting` as deprecated.

### Review checklist
- [x] Feature or bugfix MUST have appropriate tests (unit, integration)
- [x] Make sure each commit and the PR mention the Issue number or JIRA reference
- [ ] Add CHANGELOG entry for user facing changes

### Custom CI job configuration (optional)
- [ ] Run unit tests
- [ ] Run integration tests
- [x] Run smoke tests

### CODE REVIEW

File: DatadogCrashReporting.podspec [hunk 1]
```diff
@@ -0,0 +1,26 @@
+Pod::Spec.new do |s|
+  s.name         = "DatadogCrashReporting"
+  s.version      = "1.20.0"
```
Comment (maciejburda) [inline DatadogCrashReporting.podspec:3]: 1.21.0 maybe?

File: DatadogCrashReporting.podspec [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  
+  s.homepage     = "https://www.datadoghq.com"
+  s.social_media_url   = "https://twitter.com/datadoghq"
+
+  s.license            = { :type => "Apache", :file => 'LICENSE' }
+  s.authors            = { 
+    "Maciek Grzybowski" => "maciek.grzybowski@datadoghq.com",
+    "Mert Buran" => "mert.buran@datadoghq.com",
+    "Maxime Epain" => "maxime.epain@datadoghq.com"
+  }
```
Comment (maciejburda) [inline DatadogCrashReporting.podspec:14]: We should probably remove Mert and add @ganeshnj

File: DatadogCrashReporting.podspec (hunk 1 above)
Comment (maxep) [reply DatadogCrashReporting.podspec:3]: It will be updated when merging `develop` to `feature/v2` 👍

Comment (maxep) [reply DatadogCrashReporting.podspec:3]: Done in #1338

**Review by maciejburda — APPROVED** [review]

**Pull request merged**